tests/test_optimizer.py

```diff
@@ -1405,15 +1405,17 @@ def test_optimize_coil_currents(DummyCoilSet):
         coil.current = current / coils.num_coils
 
     objective = ObjectiveFunction(QuadraticFlux(eq=eq, field=coils, vacuum=True))
-    constraints = FixSumCoilCurrent(coils)
+    constraints = LinkingCurrentConsistency(eq, coils, eq_fixed=True)
     optimizer = Optimizer("lsq-exact")
-    [coils_opt], _ = optimizer.optimize(
-        things=coils,
-        objective=objective,
-        constraints=constraints,
-        verbose=2,
-        copy=True,
-    )
+    with warnings.catch_warnings():
```

Comment:
Do we understand why this constraint is being flagged as "incompatible"? This is like the simplest possible linear constraint, so I'm worried if this doesn't work without warnings.

Reply:
The error is like 1e-9 which is about what we'd expect given the currents are ~MA